### PR TITLE
Develop release2.0.10

### PR DESCRIPTION
**Pull request by asogwachiuba**
What is in this PR:
Merge completed

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -9,13 +10,24 @@ import 'package:ibloov/flavors.dart';
 import 'package:onesignal_flutter/onesignal_flutter.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:places_service/places_service.dart';
+import 'package:flutter/foundation.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  F.appFlavor = Flavor.stage;
+  F.appFlavor = Flavor.prod;
```
Comment (divincoder) [inline lib/main.dart:17]: you can change to stage

**Review by divincoder — APPROVED** [review]
Looks good. resolve comments and merge

File: lib/features/search/view/search_view/search_viewmodel.dart [hunk 2]
```diff
@@ -146,8 +146,8 @@ class SearchViewModel extends IbloovViewModel {
     Future.delayed(Duration(seconds: 3), () {
       getEvents();
       getEventCategories();
-      getPlacesCategories();
-      fetchExplorePlaces();
+      // getPlacesCategories();
```
[the remark this answers is not in the document]
Comment (asogwachiuba) [reply lib/features/search/view/search_view/search_viewmodel.dart:149]: It was duplicated, so i hid one

File: lib/features/search/widgets/search_bar.dart [hunk 3]
```diff
@@ -147,8 +147,8 @@ class SearchBar extends ViewModelWidget<SearchViewModel> {
                     EventCategory(sId: '0', name: 'All');
                 return;
               }
-              viewModel.fetchExplorePlaces();
-              viewModel.selectedPlacesCategory = viewModel.placeCategories[0];
+              // viewModel.fetchExplorePlaces();
```
[the remark this answers is not in the document]
Comment (asogwachiuba) [reply lib/features/search/widgets/search_bar.dart:150]: It was duplicated, so i commented one out

File: lib/main.dart (hunk 1 above)
Comment (asogwachiuba) [reply lib/main.dart:17]: Okay

**Pull request merged**